crates/zed/src/app_menus.rs

```diff
@@ -1,9 +1,11 @@
+use collab_ui::collab_panel;
 use gpui::{Menu, MenuItem, OsAction};
+use terminal_view::terminal_panel;
 
 pub fn app_menus() -> Vec<Menu<'static>> {
     use zed_actions::Quit;
 
-    vec![
+    let vec = vec![
```

Comment:
Why do we need this intermediate binding?

Reply:
This is a mistake.